README.md

```diff
@@ -13,12 +13,12 @@ It uses [Apollo Client](https://www.apollographql.com) and [Vue Apollo](https://
 quasar ext add @quasar/apollo@next
 ```
 
-**Note:** You need to use the `@next` tag until the final version of v2 of Quasar is released. At that point, we'll be moving v2 of the Apollo AE to the "latest" version and you will be able to install it without the `@next` tag. At that point, in order to install the older versions of the Apollo AE, you will need to add the version tag. This version will also stay in beta until Vue-Apollo is final (out of alpha or beta). 
+**Note:** You need to use the `@next` tag until the final version of v2 of Quasar is released. At that point, we'll be moving v2 of the Apollo AE to the "latest" version and you will be able to install it without the `@next` tag. At that point, in order to install the older versions of the Apollo AE, you will need to add the version tag. This version will also stay in beta until Vue-Apollo is final (out of alpha or beta).
 
 Quasar CLI will retrieve the extension from NPM
 ([@quasar/quasar-app-extension-apollo](https://www.npmjs.com/package/@quasar/quasar-app-extension-apollo))
 
-The extension will add a directory `src/extensions/apollo`.
+The extension will add a configuration file into `src/apollo` and a boot file, register the latter into `quasar.conf.js > boot`.
```

Comment:
Could you please update this sentence to the following: 

The extension will add a configuration file into `src/apollo`, also a boot file, and it will register the latter into `quasar.conf.js > boot`.

Thanks. 

Scott

Reply:
I removed the auto-registering of the boot file. If you allow the dev to change the boot file code at will, you must allow him to choose the order of execution too, so a manual registration is needed.

Maybe I should reword it to make it clearer the dev needs to manually register it?

Reply:
Sure. I was just concerned with the grammar mainly though. :)

Scott

Reply:
Is it ok as I put it now?